src/main/java/software/amazon/cloudformation/proxy/ResourceHandlerRequest.java

```diff
@@ -43,9 +43,9 @@
     private String awsPartition;
     private String logicalResourceIdentifier;
     private String nextToken;
-    private Boolean snapshotRequested;
-    private Boolean rollback;
-    private Boolean driftable;
+    private boolean snapshotRequested;
```

Comment:
I would suggest don't change the property type as it will cause method change and it will cause compile error.

As suggest in https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/issues/341#issuecomment-818960517,   we can provide extra helper method, such as `isSnapshotRequested` to simplify it.

Reply:
@wbingli I have already taken care of that change and changed the references as well. And the pr got build is  successful. Do yoy want me to revert the change and provide helper method ?

Reply:
The point is that anyone who uses this class will be broken.  The java plugin can build successful doesn't mean it won't break others.  Therefore, I suggest to add helper method in the class instead of change the field type.